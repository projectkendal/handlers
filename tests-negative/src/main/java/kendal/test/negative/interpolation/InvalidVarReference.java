package kendal.test.negative.interpolation;

public class InvalidVarReference {

    void methodReferingToMissingVar() {
        System.out.println(+"${missingVariable}");
    }
}
