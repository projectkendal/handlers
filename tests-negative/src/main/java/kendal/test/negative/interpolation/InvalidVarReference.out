InvalidVarReference.java:1:1: compiler.err.cant.resolve.location: kindname.variable, missingVariable, , , (compiler.misc.location: kindname.class, kendal.test.negative.interpolation.InvalidVarReference, null)
1 error